content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -659,6 +674,66 @@ there is no automatic Job restart once the Job status is `type: Failed`.
 That is, the Job termination mechanisms activated with `.spec.activeDeadlineSeconds`
 and `.spec.backoffLimit` result in a permanent Job failure that requires manual intervention to resolve.
 
+### Terminal Job conditions
+
+A Job has two possible terminal states, each of which has a corresponding Job
+condition:
+* Succeeded:  Job condition `Complete`
+* Failed: Job condition `Failed`.
+
+The possible reasons for a Job failure:
+- the number of pod failures exceeded the specified `.spec.backoffLimit`;
+  see also [Pod backoff failure policy](#pod-backoff-failure-policy)
+- the Job runtime exceeded the specified `.spec.activeDeadlineSeconds`
+- the Job using `.spec.backoffLimitPerIndex` has some indexes failed;
+  see also [Backoff limit per index](#backoff-limit-per-index)
+- the number of failed indexes in the Job has exceeded the specified
+  `spec.maxFailedIndexes`; see also [Backoff limit per index](#backoff-limit-per-index)
+- a failed Pod matches a rule in `.spec.podFailurePolicy`
+   with the `FailJob` action; see also [Pod failure policy](#pod-failure-policy) for more details on how the policy rules can affect evaluation of failures
+
+The possible reasons for a Job success:
+- the number of succeeded pods reached the specified `.spec.completions`
+- the criteria specified by the `.spec.successPolicy` are met;
+  see also [Success policy](#success-policy)
+
+Since v1.31 the Job controller delays the addition of the terminal conditions
```

Comment:
```suggestion
Since v1.31 the Job controller delays the addition of the terminal conditions `Complete` or `Failed`
```

I think it wouldn't hurt to reiterate the conditions.

Reply:
sure, done